src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/InbreedingCoeffUnitTest.java

```diff
@@ -0,0 +1,237 @@
+    }
+
+    @Test
+    public void testLargeCohorts() {
+
+        final List<Genotype> allGTs = new ArrayList<>();
+        final int numHomRefGTs = 1000000;
+        for ( int i = 0; i < numHomRefGTs; i++ )
+            allGTs.add(makeG("ref" + i, Aref, Aref, homRefPLs));
+
+        allGTs.add(makeG("het0", Aref, T, hetPLs));
+        int numHetGTs = 1;
+
+        final VariantContext singleton = makeVC("singleton", Arrays.asList(Aref, T), allGTs.toArray(new Genotype[allGTs.size()]));
+        final double ICsingleton = new InbreedingCoeff().calculateIC(singleton, singleton.getGenotypes()).getRight();
+
+        for ( int i = numHetGTs; i < 100; i++ ) {
+            allGTs.add(makeG("het" + i, Aref, T, hetPLs));
+            numHetGTs++;
+        }
+
+        final VariantContext hundredton = makeVC("hundredton", Arrays.asList(Aref, T), allGTs.toArray(new Genotype[allGTs.size()]));
+        final double IChundredton = new InbreedingCoeff().calculateIC(hundredton, hundredton.getGenotypes()).getRight();
+
+        Assert.assertTrue(Math.abs(ICsingleton) < Math.abs(IChundredton), String.format("singleton=%f hundredton=%f", ICsingleton, IChundredton));
+
+        for ( int i = numHetGTs; i < numHomRefGTs; i++ )
+            allGTs.add(makeG("het" + i, Aref, T, hetPLs));
+
+        final VariantContext common = makeVC("common", Arrays.asList(Aref, T), allGTs.toArray(new Genotype[allGTs.size()]));
+        final double ICcommon = new InbreedingCoeff().calculateIC(common, common.getGenotypes()).getRight();
+
+        Assert.assertTrue(Math.abs(IChundredton) < Math.abs(ICcommon), String.format("hundredton=%f common=%f", IChundredton, ICcommon));
+    }
+
+    @Test
+    public void testAllHetsForLargeCohorts() {
+
+        final int numGTs = 1000000;
+
+        final List<Genotype> singletonGTs = new ArrayList<>();
+        for ( int i = 0; i < numGTs; i++ )
+            singletonGTs.add(makeG("ref" + i, Aref, Aref, homRefPLs));
+
+        singletonGTs.add(makeG("het0", Aref, T, hetPLs));
+
+        final VariantContext singleton = makeVC("singleton", Arrays.asList(Aref, T), singletonGTs.toArray(new Genotype[singletonGTs.size()]));
+        final double ICsingleton = new InbreedingCoeff().calculateIC(singleton, singleton.getGenotypes()).getRight();
+
+        final List<Genotype> allHetGTs = new ArrayList<>();
+        for ( int i = 0; i < numGTs; i++ )
+            allHetGTs.add(makeG("het" + i, Aref, T, hetPLs));
+
+        final VariantContext allHet = makeVC("allHet", Arrays.asList(Aref, T), allHetGTs.toArray(new Genotype[allHetGTs.size()]));
+        final double ICHets = new InbreedingCoeff().calculateIC(allHet, allHet.getGenotypes()).getRight();
+
+        Assert.assertTrue(Math.abs(ICsingleton) < Math.abs(ICHets), String.format("singleton=%f allHets=%f", ICsingleton, ICHets));
+    }
```

Comment:
I think it would be good to add a test where a pedigree is passed in (although there are no pedigrees in production right now).  Maybe two tests: one test with 10 samples, but 3 are a trio (result: no annotation, fewer than 10 founders); one test with 20 samples, >=3 are in trios (result: InbreedingCoeff is calculated, but does not use the children of the trios)